src/main/java/com/axlabs/neo/grantshares/GrantSharesGov.java

```diff
@@ -472,7 +497,11 @@ public static Object[] execute(int id) {
         ProposalData data = (ProposalData) new StdLib().deserialize(proposalData.get(id));
         ProposalVotes votes = (ProposalVotes) new StdLib().deserialize(proposalVotes.get(id));
         int voteCount = votes.approve + votes.abstain + votes.reject;
-        if (voteCount * 100 / Storage.getInt(getReadOnlyContext(), MEMBERS_COUNT_KEY) < data.quorum) {
+        // first check for old-style proposals that don't have a quorumVotes value
+        if (proposal.quorumVotes == 0 &&
+                voteCount * 100 / Storage.getInt(getReadOnlyContext(), MEMBERS_COUNT_KEY) < data.quorum) {
+            Helper.abort("execute" + ": " + "Proposal not voted on");
+        } else if (voteCount < proposal.quorumVotes) {
```

Comment:
```suggestion
        if (voteCount < proposal.quorumVotes) {
```

question: this was added before you added the migration code, right? I don't think we need this part any longer since the migration updates all proposals to the new structure. So, all proposals will have the "new" style.

Reply:
yeah, that's correct. it's no longer required with the migration